### ISSUE TITLE
[accounts] Switching Accounts don't update view in Folders.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#318]
```
What steps will reproduce the problem?
1. Set up additional IMAP Account (Settings > Account administration)
2. Go to home, now you have 2 Accounts in dropdown and the corresponding IMAP 
Folders.
3. Switch Account
4. Folders remains from first Account (so you get errors when you click a 
Folder that does not exist.)
5. After switching to an another account, the listed mail address is wrong with 
2 times the same address and the principal missing.


Expected output would be that the Folders list would be reloaded with the right 
folders from the second account. (if you click down left "manage folders" and 
after that you click on the roundcube logo, it works like expected.


What version of the product are you using? On what operating system?
Roundcube 0.7.2
Firefox 11
Fedora or win7 


Please provide any additional information below.

My folder settings is by default:

// store draft message is this mailbox
// leave blank if draft messages should not be stored
// NOTE: Use folder names with namespace prefix (INBOX. on Courier-IMAP)
$rcmail_config['drafts_mbox'] = 'Drafts';

// store spam messages in this mailbox
// NOTE: Use folder names with namespace prefix (INBOX. on Courier-IMAP)
$rcmail_config['junk_mbox'] = 'Junk';

// store sent message is this mailbox
// leave blank if sent messages should not be stored
// NOTE: Use folder names with namespace prefix (INBOX. on Courier-IMAP)
$rcmail_config['sent_mbox'] = 'Sent';

// move messages to this folder when deleting them
// leave blank if they should be deleted directly
// NOTE: Use folder names with namespace prefix (INBOX. on Courier-IMAP)
$rcmail_config['trash_mbox'] = 'Trash';

// display these folders separately in the mailbox list.
// these folders will also be displayed with localized names
// NOTE: Use folder names with namespace prefix (INBOX. on Courier-IMAP)
$rcmail_config['default_imap_folders'] = array('INBOX', 'Drafts', 'Sent', 
'Junk', 'Trash');

// automatically create the above listed default folders on first login
$rcmail_config['create_default_folders'] = false;

// protect the default folders from renames, deletes, and subscription changes
$rcmail_config['protect_default_folders'] = true;
```

Original issue reported on code.google.com by `pm2...@gmail.com` on 31 Mar 2012 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
I get these kind of error reports frequently. I'm not able to reproduce this on 
my system. Check yourself (register account @ http://mail4us.net). If you are 
able to reproduce it there, please mail me account details (myroundcube at 
mail4us dot net). If not, it would be great to get access to your system (IMAP 
Test-Account and FTP or SSH to Roundcube Root Folder). Thanks!
```

Original comment by `rol...@liebl.ath.cx` on 31 Mar 2012 at 10:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 31 Mar 2012 at 12:47
- Changed state: **Reproducing**

**Comment by GoogleCodeExporter** [comment]
```
Ok so, I can't reproduce the problem on your system.
Another important information is: I haven't got any local imap account, all is 
in google apps or gmail.
I will try to give you an ssh access
```

Original comment by `pm2...@gmail.com` on 1 Apr 2012 at 9:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 1 Apr 2012 at 4:23
- Changed title: **[accounts] Switching Accounts don't update view in Folders.**

**Comment by GoogleCodeExporter** [comment]
```
I can't reproduce the issue.

I logged into your webmail with my main Google Account. Then I added a second 
Google Account. I can switch between both accounts without any problems. Folder 
list is updated correctly.

Did I miss anything?
```

Original comment by `rol...@liebl.ath.cx` on 10 Apr 2012 at 4:47

**Comment by GoogleCodeExporter** [comment]
```
No feedback ... closing within 48 hours.
```

Original comment by `rol...@liebl.ath.cx` on 11 Apr 2012 at 7:19

**Comment by GoogleCodeExporter** [comment]
```
I have send you a mail yesterday with an account with problem

Regards
```

Original comment by `pm2...@gmail.com` on 11 Apr 2012 at 7:55

**Comment by GoogleCodeExporter** [comment]
```
I didn't get the email. Send it again to myroundcube at mail4us dot net.
```

Original comment by `rol...@liebl.ath.cx` on 11 Apr 2012 at 9:22

**Comment by GoogleCodeExporter** [comment]
```
Fixed in account v3.2
```

Original comment by `rol...@liebl.ath.cx` on 12 Apr 2012 at 5:52
- Changed state: **Fixed**
